packages/flutter/lib/src/services/autofill.dart

```diff
@@ -0,0 +1,287 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'text_input.dart';
+
+// TODO(LongCatIsLooong): expand the list to include every predefined value.
+/// A collection of commonly used autofill hint strings on different platforms.
+///
+/// Each hint may not be supported on every platform, and may get translated to
+/// different strings on different platforms. Please refer to their documentation
+/// for what they corresponds to on each platform.
+class AutofillHints {
```

Comment:
Can we also add a few more? To me it feels like developers will immediately will need postal-code, country, email to make complete signup page.

Reply:
I was considering leaving this empty to make this PR easier to review, but that makes it harder to revert. I'll add the complete list of hints then.